scoring/score_submission.py

```diff
@@ -1,11 +1,16 @@
+import operator
 import os
 
 from absl import app
 from absl import flags
 from absl import logging
+import numpy as np
+import pandas as pd
 import scoring_utils
+from tabulate import tabulate
 
-from scoring import scoring
+from scoring import performance_profile
+from scoring.performance_profile import check_if_minimized
```

Comment:
Before I start my review, is it possible that a new file called `performance_profile` is missing?

Reply:
Ah I renamed the file and it didn't get staged apparently

Reply:
Thanks for catching that, I just added it.